haiku-apps/wordclock/wordclock-0.1.1.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="An imprecise clock"
+DESCRIPTION="WordClock is an imprecise clock that displays the time in words instead of digits"
```

Comment:
There's a full-stop missing. Also, didn't we wanted to add that it can be used as a Replicant?
When adding that, we can also point to another feature. So:
```
DESCRIPTION="WordClock is an imprecise clock that displays the time in words \
instead of digits. You can change the used font and embed the clock as a \
Replicant on the Desktop."
```